_posts/blog/2018-10-18-review-living-archive.md

```diff
@@ -0,0 +1,20 @@
+
+Access is an guiding force for this archive, but for whom is the key question. Jeannette A. Bastian states “the creation of collective memory by nations, communities, or groups of people depends upon their ability to access their own cultural heritage” (81). For Bastian, access should be a fundamental component of custody. The Living Archive takes this insightful argument in practice as an example of postcustodial archiving. The project does not include keeping the original material they tracked down, rather is invested in making the books reachable for a diverse group of people –linguists, teachers, students, researches, as their website guide shows. But more important is its redress to the communities that created the books, because the digital collection liberated their own cultural heritage from depending on the aftermath power infrastructures and custody of a discontinued bilingual educational program. 
+
+There is evidence of an accountability and ethical responsibility to the indigenous peoples of the Northern Territory from the first time one enters the website. A notice that reads “Respecting ownership” is at its gate. It indicates that the contents belong to the “Aboriginal language owners, creators of the materials and their descendants.” The notice also serves as a warning to the Aboriginal and Torres Strait Islander users about the inclusion of “names, images, and voices of people who have passed away”. Once entered the website, browsing is very easy. Apart from the mentioned interactive map organized by places and languages, there is a menu bar with different categories for searching purposes: “Place”, “Language”, “People” (authors and illustrators), and “Books”. After a selection has been made, the new page shows the covers of the books. Some of them are very colorful, and most of them have a naïve esthetic meant for attracting the attention of children. The metadata is very thorough –title, author, illustrator, place, year of publication, abstract, description/genre, and others–, and it is followed by a “Right Statement”. This announcement entails a decolonial practice, that of contacting the creators for permission to publish, but also a call for users to reach out if they feel the book should not be shown. All the documents are available for exploring in PDF format, and for downloads. The website has a Creative Common License that privileges the sharing of the material.
```

Comment:
Corrections:
* s1: this phrasing is ambiguous. There is evidence that the project is holding itself accountable? That they are fulfilling an ethical responsibility?
* available for *download*
* once *entering* the website
* *Rights* statement
* "permission to publish, *and of calling on* users to reach out if..."